cluster-autoscaler/cloudprovider/aws/ec2_instance_types/gen.go

```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```

Comment:
@AustinSiu looks like the CI's Go version doesn't like this change. Best to remove it.